components/automate-cli/pkg/verifyserver/server/api/v1/status.go

```diff
@@ -4,21 +4,24 @@ import (
 	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/constants"
 	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
 	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/response"
+	"github.com/chef/automate/lib/version"
 	"github.com/gofiber/fiber/v2"
 )
 
 func (h *Handler) GetStatus(c *fiber.Ctx) error {
 	services, err := h.StatusService.GetServices()
 	if err != nil {
 		return c.JSON(response.BuildSuccessResponse(&models.StatusDetails{
-			Status:   constants.OK,
-			Services: &[]models.ServiceDetails{},
-			Error:    err.Error(),
+			Status:     constants.OK,
+			Services:   &[]models.ServiceDetails{},
+			CliVersion: version.BuildTime,
```

Comment:
Please add a demo video, showing this response from the status API

Reply:
done